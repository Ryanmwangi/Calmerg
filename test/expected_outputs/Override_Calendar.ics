BEGIN:VCALENDAR
PRODID:-//Your Product ID//EN
VERSION:2.0
NAME:Override Calendar
BEGIN:VEVENT
UID:20231225T000000-001@example.com
SUMMARY:Override Event
DTSTART:20231225T000000
DTEND:20231226T000000
END:VEVENT
END:VCALENDAR