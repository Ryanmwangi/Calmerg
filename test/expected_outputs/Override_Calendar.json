{
  "linkGroupName": "Override Calendar",
  "calendars": [
    {
      "url": "C:\\Users\\user\\OneDrive\\Desktop\\Internship_tasks\\final calmerg\\test\\test_calendars\\sf_public_holidays.ics",
      "prefix": "Override Event",
      "override": true
    }
  ]
}